docs/docs/producers.md

```diff
@@ -220,6 +220,23 @@ Set the output queue in the `config.properties` by setting the `sqs_queue_uri` p
 
 The producer uses the [AWS SQS SDK](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/sqs/AmazonSQSClient.html).
 
+# SNS
+***
+
+## AWS Credentials
+You will need to obtain an IAM user that has the permission to access the SNS topic. The SNS producer also uses [DefaultAWSCredentialsProviderChain](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/DefaultAWSCredentialsProviderChain.html) to get AWS credentials.
+
+See the [AWS docs](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html) on how to setup the IAM user with the Default Credential Provider Chain.
+
+In case you need to set up a different region also along with credentials then default one, see the [AWS docs](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/setup-credentials.html#setup-credentials-setting-region).
+
+## Options
+Set the topic arn in the `config.properties` by setting the `sns_topic` property to the topic name. FIFO topics should have a `.fifo` suffix. 
+
+Optionally, you can enable `sns_table_attribute` and/or `sns_database_attribute` values to have maxwell attach `table` and`database` attributes to the message for subscription filtering
+
+The producer uses the [AWS SNS SDK](http://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/sns/AmazonSNSClient.html).
+
```

Comment:
is there further configuration for the SNS client library possible?  If so it'd be nice to drop hints as to where to find more config info (rather than just pointing them to the top level SDK docs)

Reply:
First, I'll admit it was a copy - paste from the SQS producer, as they are fairly similar. :/

Regarding configuration, I'm not sure there's a whole lot of "configuration" on the client library itself. It pulls credentials from the environment, then you give it a topic and message for the `publishAsync()` call. I'll poke around to see if I can find something more appropriate

Reply:
If there's nothing more to configure, no big deal.